src/app/features/auth/login/login.component.ts

```diff
@@ -0,0 +1,88 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { AbstractControl, FormGroup, NonNullableFormBuilder, ReactiveFormsModule, Validators } from "@angular/forms";
+import { Router } from "@angular/router";
+import { AuthService } from "../services/auth.service";
+import { StorageService } from "../../../core/services/storage.service";
+import { Store } from "@ngxs/store";
+import { LoginAction, LoginTokenAction } from "@src/app/state/auth.action";
+import { Token } from "@src/app/shared/models/token";
+import { take } from "rxjs";
+
+@Component({
+  selector: 'login',
+  standalone: true,
+  imports: [CommonModule, ReactiveFormsModule],
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class LoginComponent implements OnInit {
+
+  public form: FormGroup;
+  public isSubmitted = false;
+
+  constructor(
+    private readonly fb: NonNullableFormBuilder,
+    private readonly router: Router,
+    private readonly authService: AuthService,
+    private readonly localStorage: StorageService,
+    private readonly store: Store) {
+    this.form = this.fb.group({
+      email: ['', {
+        validators: [Validators.required, Validators.email]
+      }],
+      password: ['', [Validators.required, Validators.minLength(6)]],
```

Comment:
On line 33 and 35 you're doing the same thing – declaring validators, but differently: providing an object on line 33 and an array – on line 35. Keep the code consistent, use the same approach everywhere.